src/Util/Test.php

```diff
@@ -517,7 +518,7 @@ private static function getDataFromDataProviderAnnotation($docComment, $classNam
                     $data = $dataProviderMethod->invoke($object, $methodName);
                 }
 
-                if ($data instanceof Iterator) {
+                if ($data instanceof Traversable) {
                     $data = \iterator_to_array($data);
```

Comment:
> And my proposal is deeper - to allow dataProvider not only work properly with Traversable, but all iterable.

I think I don't understand. `Traversable` and `array` are all `iterable` possibilities, so IMO this change is good enough. Is there something what is skipped?